src/Swashbuckle.AspNetCore.SwaggerGen/DependencyInjection/DocumentProvider.cs

```diff
@@ -24,15 +26,28 @@ internal class DocumentProvider : IDocumentProvider
         private readonly SwaggerGeneratorOptions _generatorOptions;
         private readonly SwaggerOptions _options;
         private readonly IAsyncSwaggerProvider _swaggerProvider;
+        private readonly ISwaggerDocumentSerializer _swaggerDocumentSerializer;
 
         public DocumentProvider(
             IOptions<SwaggerGeneratorOptions> generatorOptions,
             IOptions<SwaggerOptions> options,
-            IAsyncSwaggerProvider swaggerProvider)
+            IAsyncSwaggerProvider swaggerProvider
+            ) : this(generatorOptions, options, swaggerProvider, null)
```

Comment:
```suggestion
            IAsyncSwaggerProvider swaggerProvider)
            : this(generatorOptions, options, swaggerProvider, null)
```

Reply:
If I put the closing bracket on this location here, I think it should also go to that location in the other constructor